fantasy_postseason_challenge/templates/register.html

```diff
@@ -9,6 +9,8 @@ <h1>Register</h1>
         <form action="" method="post">
             <label for="username">username: </label><br>
             <input type="text" id="username" name="username"><br>
+            <label for="display_name">display name: </label><br>
```

Comment:
change "display name:" to "Your name:" as the text displayed to the user. Also, capitalize 'Username' and 'Password' display texts